shaders/screen_effects/Gameboy.shader

```diff
@@ -0,0 +1,22 @@
+shader_type canvas_item;
+
+const float color_count = 4.0f;
+const int pixel_count = 512;
+
+const vec3 palette[4] = {
+    vec3(0.0f, 63.0f, 0.0f) / 255.0f,
+    vec3(47.0f, 114.0f, 32.0f) / 255.0f,
+    vec3(140.0f, 191.0f, 10.0f) / 255.0f,
+    vec3(161.0f, 206.0f, 10.0f) / 255.0f
+};
+
+void fragment()
+{
+    vec2 grid_UV = (vec2(ivec2( SCREEN_UV * float(pixel_count) ))+0.5f)/float(pixel_count);
```

Comment:
The spacing is inconsistent on this line, spaces around parenthesis should be removed and spaces should be around operators. This same issue is in the other shader files as well.

`(vec2(ivec2(SCREEN_UV * float(pixel_count))) + 0.5f) / float(pixel_count);`